src/test/java/com/wikia/webdriver/elements/mercury/old/PortableInfoboxObject.java

```diff
@@ -55,6 +55,9 @@
   @FindBy(css = ".image-collection-actions .action-next")
   private WebElement nextImageArrow;
 
+  @FindBy(css = ".pi-image:nth-child(2) .action-next")
+  private WebElement secondNextImageArrow;
```

Comment:
how about this:
`@FindBy(css = ".image-collection-actions .action-next")
private List<WebElement> nextImageArrows`

and then instead of separate method for every click on next button, one could be parametrised with index?